### PR TITLE
Improve dependant overlay messages

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Bug fix? | no 
| New feature? | no 
| BC breaks? | no 
| Deprecations? | no  
| Fixed tickets | fixes # 
| Related issues/PRs | # 
| License | MIT
| Documentation PR | sulu/sulu-docs# 

#### What's in this PR?

Improve dependant overlay messages.

#### Why?

The exist dependant messages are very unspecific and can confuse Users so we go with a more specific message for the overlay and make it possible to control it from PHP instead of hardcode it in the JS.

### CODE REVIEW

File: src/Sulu/Bundle/MediaBundle/Resources/translations/admin.de.json [hunk 1]
```diff
@@ -79,6 +79,8 @@
     "sulu_media.point": "Punkt",
     "sulu_media.rectangle": "Rechteck",
     "sulu_media.parent_collection": "Elternkollektion",
+    "sulu_media.delete_collection_warning_title": "{1}Eine zusätzliche abhängige Kollektion oder Medium löschen|]1,Inf[%count% zusätzliche abhängige Kollektionen oder Medien löschen",
```
Comment (alexander-schranz) [inline src/Sulu/Bundle/MediaBundle/Resources/translations/admin.de.json]: This is a little bit to long as "title" what would you suggest here @chirimoya

File: src/Sulu/Bundle/AdminBundle/Resources/translations/admin.de.json [hunk 2]
```diff
@@ -80,8 +80,8 @@
     "sulu_admin.delete_selection_warning_text": "Diese Operation löscht {count} {count, plural, =1 {Datensatz} other {Datensätze}} und kann nicht rückgängig gemacht werden. Wollen Sie wirklich fortfahren?",
     "sulu_admin.item_not_deletable": "Eintrag nicht löschbar",
     "sulu_admin.delete_linked_abort_text": "Dieser Eintrag kann nicht gelöscht werden, solange er noch von anderen Einträgen referenziert wird. Bitte löschen Sie die folgenden referenzierenden Einträge oder entfernen sie die Relation",
-    "sulu_admin.delete_dependants_warning_title": "{count} abhängige Elemente löschen",
-    "sulu_admin.delete_dependants_warning": "Sind Sie sich sicher, dass Sie {count} abhängige Elemente löschen möchten?",
+    "sulu_admin.delete_dependants_warning_title": "{1}Ein zusätzliche abhängiges Element löschen|]1,Inf[%count% zusätzliche abhängige Elemente löschen",
```
Comment (chirimoya) [inline src/Sulu/Bundle/AdminBundle/Resources/translations/admin.de.json]:
```suggestion
    "sulu_admin.delete_dependants_warning_title": "{1}Ein abhängiges Element löschen?|]1,Inf[%count% abhängige Elemente löschen?",
```

File: src/Sulu/Bundle/AdminBundle/Resources/translations/admin.de.json [hunk 3]
```diff
@@ -80,8 +80,8 @@
...
+    "sulu_admin.delete_dependants_warning": "{1}Sind Sie sich sicher, dass Sie ein zusätzlich abhängiges Element ebenfalls löschen möchten?|]1,Inf[Sind Sie sich sicher, dass Sie %count% abhängige Elemente ebenfalls löschen möchten?",
```
Comment (chirimoya) [inline src/Sulu/Bundle/AdminBundle/Resources/translations/admin.de.json]:
```suggestion
    "sulu_admin.delete_dependants_warning": "{1}Sind Sie sich sicher, dass Sie zusätzlich ein abhängiges Element löschen möchten?|]1,Inf[Sind Sie sich sicher, dass Sie zusätzlich %count% abhängige Elemente löschen möchten?",
```

File: src/Sulu/Bundle/AdminBundle/Resources/translations/admin.en.json [hunk 4]
```diff
@@ -81,8 +81,8 @@
     "sulu_admin.delete_selection_warning_text": "This operation deletes {count} {count, plural, =1 {item} other {items}} and cannot be undone. Do you really wish to proceed?",
     "sulu_admin.item_not_deletable": "Item cannot be deleted",
     "sulu_admin.delete_linked_abort_text": "This item cannot be deleted as long as it has sub items. Delete the following sub items or remove the relation:",
-    "sulu_admin.delete_dependants_warning_title": "Delete {count} dependant elements",
-    "sulu_admin.delete_dependants_warning": "Are you sure, that you want to delete {count} dependant elements?",
+    "sulu_admin.delete_dependants_warning": "{1}Are you sure, that you want to delete one additional dependant element?|]1,Inf[Are you sure, that you want to delete %count% additional dependant elements?",
```
Comment (chirimoya) [inline src/Sulu/Bundle/AdminBundle/Resources/translations/admin.en.json]:
```suggestion
    "sulu_admin.delete_dependants_warning": "{1}Are you sure that you also want to delete one dependant element?|]1,Inf[Are you sure that you also want to delete %count% dependant elements?",
```

File: src/Sulu/Bundle/AdminBundle/Resources/translations/admin.en.json [hunk 5]
```diff
@@ -81,8 +81,8 @@
...
+    "sulu_admin.delete_dependants_warning_title": "{1}Delete one additional dependant element|]1,Inf[Delete %count% additional dependant elements",
```
Comment (chirimoya) [inline src/Sulu/Bundle/AdminBundle/Resources/translations/admin.en.json]:
```suggestion
    "sulu_admin.delete_dependants_warning_title": "{1}Delete one dependant element?|]1,Inf[Delete %count% dependant elements?",
```

File: src/Sulu/Bundle/CategoryBundle/Resources/translations/admin.de.json [hunk 6]
```diff
@@ -11,6 +11,8 @@
     "sulu_category.no_category_selected": "Keine Kategorie ausgewählt",
     "sulu_category.single_category_selection_overlay_title": "Kategorie auswählen",
     "sulu_category.key_assigned_to_other_category": "Der Schlüssel \"{key}\" ist bereits einer anderen Kategorie zugewiesen.",
+    "sulu_category.delete_category_warning_title": "{1}Zusätzliche abhängige Kategorie löschen|]1,Inf[%count% zusätzliche abhängige Kategorien löschen",
```
Comment (chirimoya) [inline src/Sulu/Bundle/CategoryBundle/Resources/translations/admin.de.json]:
```suggestion
    "sulu_category.delete_category_warning_title": "{1}Eine Unterkategorie löschen?|]1,Inf[%count% Unterkategorien löschen?",
```

File: src/Sulu/Bundle/CategoryBundle/Resources/translations/admin.de.json [hunk 7]
```diff
@@ -11,6 +11,8 @@
...
+    "sulu_category.delete_category_warning": "{1}Sind Sie sich sicher, dass Sie eine zusätzliche abhängige Kategorie ebenfalls löschen möchten?|]1,Inf[Sind Sie sich sicher, dass Sie %count% zusätzliche abhängige Kategorie löschen möchten?",
```
Comment (chirimoya) [inline src/Sulu/Bundle/CategoryBundle/Resources/translations/admin.de.json]:
```suggestion
    "sulu_category.delete_category_warning": "{1}Sind Sie sich sicher, dass Sie zusätzlich eine Unterkategorie löschen möchten?|]1,Inf[Sind Sie sich sicher, dass Sie zusätzlich %count% Unterkategorien löschen möchten?",
```

File: src/Sulu/Bundle/CategoryBundle/Resources/translations/admin.en.json [hunk 8]
```diff
@@ -11,6 +11,8 @@
     "sulu_category.no_category_selected": "No category selected",
     "sulu_category.single_category_selection_overlay_title": "Choose category",
     "sulu_category.key_assigned_to_other_category": "The key \"{key}\" is already assigned to another category.",
+    "sulu_category.delete_category_warning_title": "{1}Delete one additional category|]1,Inf[Delete %count% additional categories",
```
Comment (chirimoya) [inline src/Sulu/Bundle/CategoryBundle/Resources/translations/admin.en.json]:
```suggestion
    "sulu_category.delete_category_warning_title": "{1}Delete one subcategory?|]1,Inf[Delete %count% subcategories?",
```

File: src/Sulu/Bundle/CategoryBundle/Resources/translations/admin.en.json [hunk 9]
```diff
@@ -11,6 +11,8 @@
...
+    "sulu_category.delete_category_warning": "{1}Are you sure, that you want to delete one additional category?|]1,Inf[Are you sure, that you want to delete %count% categories?",
```
Comment (chirimoya) [inline src/Sulu/Bundle/CategoryBundle/Resources/translations/admin.en.json]:
```suggestion
    "sulu_category.delete_category_warning": "{1}Are you sure that you also want to delete one subcategory?|]1,Inf[Are you sure that you also want to delete %count% subcategories?",
```

File: src/Sulu/Bundle/PageBundle/Resources/translations/admin.de.json [hunk 10]
```diff
@@ -63,6 +63,8 @@
     "sulu_page.add_schedule": "Zeitfenster hinzufügen",
     "sulu_page.schedules_label": "Zeitfenster zuweisen",
     "sulu_page.schedules_description": "Der Block wird nur angezeigt, wenn mindestens einer der Bedingungen erfüllt wird.",
+    "sulu_page.delete_page_warning_title": "{1}Zusätzliche abhängige Seite löschen|]1,Inf[%count% zusätzliche abhängige Seiten löschen",
```
Comment (chirimoya) [inline src/Sulu/Bundle/PageBundle/Resources/translations/admin.de.json]:
```suggestion
    "sulu_page.delete_page_warning_title": "{1}Eine Unterseite löschen?|]1,Inf[%count% Unterseiten löschen?",
```

File: src/Sulu/Bundle/PageBundle/Resources/translations/admin.de.json [hunk 11]
```diff
@@ -63,6 +63,8 @@
...
+    "sulu_page.delete_page_warning": "{1}Sind Sie sich sicher, dass Sie eine zusätzliche abhängige Seite ebenfalls löschen möchten?|]1,Inf[Sind Sie sich sicher, dass Sie %count% zusätzliche abhängige Seiten löschen möchten?",
```
Comment (chirimoya) [inline src/Sulu/Bundle/PageBundle/Resources/translations/admin.de.json]:
```suggestion
    "sulu_page.delete_page_warning": "{1}Sind Sie sich sicher, dass Sie zusätzlich eine Unterseiten löschen möchten?|]1,Inf[Sind Sie sich sicher, dass Sie zusätzlich %count% Unterseiten löschen möchten?",
```

File: src/Sulu/Bundle/PageBundle/Resources/translations/admin.en.json [hunk 12]
```diff
@@ -63,6 +63,8 @@
     "sulu_page.add_schedule": "Add schedule",
     "sulu_page.schedules_label": "Assign time schedule",
     "sulu_page.schedules_description": "The block will only be visible if at least one condition is valid.",
+    "sulu_page.delete_page_warning_title": "{1}Delete one additional page|]1,Inf[Delete %count% additional pages",
```
Comment (chirimoya) [inline src/Sulu/Bundle/PageBundle/Resources/translations/admin.en.json]:
```suggestion
    "sulu_page.delete_page_warning_title": "{1}Delete one subpage?|]1,Inf[Delete %count% subpages?",
```

File: src/Sulu/Bundle/PageBundle/Resources/translations/admin.en.json [hunk 13]
```diff
@@ -63,6 +63,8 @@
...
+    "sulu_page.delete_page_warning": "{1}Are you sure, that you want to delete one additional page?|]1,Inf[Are you sure, that you want to delete %count% additional pages?",
```
Comment (chirimoya) [inline src/Sulu/Bundle/PageBundle/Resources/translations/admin.en.json]:
```suggestion
    "sulu_page.delete_page_warning": "{1}Are you sure that you also want to delete one subpage?|]1,Inf[Are you sure that you also want to delete %count% subpages?",
```

File: src/Sulu/Bundle/MediaBundle/Resources/translations/admin.de.json (hunk 1 above)
Comment (chirimoya) [inline src/Sulu/Bundle/MediaBundle/Resources/translations/admin.de.json]:
```suggestion
    "sulu_media.delete_collection_warning_title": "{1}Ein Unterelement löschen?|]1,Inf[%count% Unterelemente löschen?",
```

File: src/Sulu/Bundle/MediaBundle/Resources/translations/admin.de.json [hunk 14]
```diff
@@ -79,6 +79,8 @@
...
+    "sulu_media.delete_collection_warning": "{1}Sind Sie sich sicher, dass Sie ein zusätzlich abhängige Kollektion oder Medium ebenfalls löschen möchten?|]1,Inf[Sind Sie sich sicher, dass Sie %count% zusätzlich abhängige Kollektionen oder Medien ebenfalls löschen möchten?",
```
Comment (chirimoya) [inline src/Sulu/Bundle/MediaBundle/Resources/translations/admin.de.json]:
```suggestion
    "sulu_media.delete_collection_warning": "{1}Sind Sie sich sicher, dass Sie zusätzlich einen Unterordner oder ein Medium löschen möchten?|]1,Inf[Sind Sie sich sicher, dass Sie zusätzlich %count% Unterordner beziehungsweise Medien löschen möchten?",
```

File: src/Sulu/Bundle/MediaBundle/Resources/translations/admin.en.json [hunk 15]
```diff
@@ -79,6 +79,8 @@
     "sulu_media.point": "Point",
     "sulu_media.rectangle": "Rectangle",
     "sulu_media.parent_collection": "Parent Collection",
+    "sulu_media.delete_collection_warning_title": "{1}Delete one additional collection or media|]1,Inf[Delete %count% additional collection or media",
```
Comment (chirimoya) [inline src/Sulu/Bundle/MediaBundle/Resources/translations/admin.en.json]:
```suggestion
    "sulu_media.delete_collection_warning_title": "{1}Delete one subelement?|]1,Inf[Delete %count% subelements?",
```

File: src/Sulu/Bundle/MediaBundle/Resources/translations/admin.en.json [hunk 16]
```diff
@@ -79,6 +79,8 @@
...
+    "sulu_media.delete_collection_warning": "{1}Are you sure, that you want to delete one additional collection or media?|]1,Inf[Are you sure, that you want to delete %count% additional collection or media?",
```
Comment (chirimoya) [inline src/Sulu/Bundle/MediaBundle/Resources/translations/admin.en.json]:
```suggestion
    "sulu_media.delete_collection_warning": "{1}Are you sure that you also want to delete one subcollection or media?|]1,Inf[Are you sure that you also want to delete %count% subcollection or media?",
```

**Review by chirimoya — CHANGES_REQUESTED** [review]

**Pull request merged**